src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -340,10 +340,11 @@ private BoundExpression ConvertSwitchExpression(BoundUnconvertedSwitchExpression
             {
                 var oldCase = source.SwitchArms[i];
                 var oldValue = oldCase.Value;
+                var binder = GetRequiredBinder(oldCase.Syntax);
```

Comment:
❓ I'm not sure if this is the right way to acquire the binder. Should we capture it in the bound node instead?